src/ServiceControlInstaller.Engine/ServiceControlInstaller.Engine.csproj

```diff
@@ -38,4 +38,10 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="WorkaroundForSqlClientWindowsFormsReference" BeforeTargets="AddTransitiveFrameworkReferences">
```

Comment:
The dependencies of the sql client are a big mess but there seems to be some movement

https://github.com/dotnet/SqlClient/issues/1108

Reply:
Yeah, I'm hoping that will be resolved soon and then we can remove these workaround targets.